src/main/java/games/strategy/triplea/delegate/UndoablePlacement.java

```diff
@@ -16,12 +16,15 @@
 /**
  * Contains all the data to describe a placement and to undo it.
  */
-class UndoablePlacement extends AbstractUndoableMove {
+public class UndoablePlacement extends AbstractUndoableMove {
   private static final long serialVersionUID = -1493488646587233451L;
   final Territory m_placeTerritory;
   Territory m_producerTerritory;
   PlayerID m_player;
 
+  /**
+   * Public UndoablePlacement constructor.
```

Comment:
The constructor javadoc requirement is a bit controversial. Regardless, it should be useful.
'public' restates what is on the next line, a comment is not useful when it just restates or rephrases what is already written and compiler checked. 

I've found it's easiest to comment on constructors by stating what kind of state is or is not initialized. Here the comment maybe particularly interesting since the class state is relatively complex (noting that the variables are not private, so an unknown number of child classes can interact with those variables).